common/src/main/java/com/wynntils/models/items/encoding/type/DataTransformerType.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright © Wynntils 2023.
+ * This file is released under LGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.models.items.encoding.type;
+
+public enum DataTransformerType {
+    START_DATA_TRANSFORMER((byte) 0),
```

Comment:
Maybe make the constructor take an int and do the casting in a single line only in the constructor?

Reply:
Sure.